src/fn/edit-contacts.js

```diff
@@ -175,11 +175,15 @@ const parseExtraArgs = (projectDir, extraArgs = []) => {
   const csvFiles = (args.files || args.file || 'contact.csv')
     .split(',')
     .filter(id => id);
+  
+  const updateOfflineDocs = String(args.updateOfflineDocs).toLowerCase() === 'true'? true : false;
```

Comment:
Yes. Casting to a string takes care of that. I have added a couple of test cases for that though.

Reply:
minimist will handle the boolean check for you.  Instead of doing a match on the string. Just return `args.updateOfflineDocs`. Then the user only has to pass `--updateOfflineDocs` or omit  for false. This means you can remove your tests for checking different invalid strings being passed for the boolean check.  See the `--accept-self-signed-certs` as a bool check. 


```
  return {
    colNames,
    csvFiles,
    docDirectoryPath: path.resolve(projectDir, args.docDirectoryPath || 'json_docs'),
    updateOfflineDocs = args.updateOfflineDocs,
    force: !!args.force,
  };
```